specifyweb/frontend/js_src/lib/components/DataModel/types.ts

```diff
@@ -6499,14 +6499,14 @@ export type CollectionObjectGroup = {
   };
   readonly toOneDependent: RR<never, never>;
   readonly toOneIndependent: {
+    readonly parentcojo: CollectionObjectGroupJoin | null;
```

Comment:
https://github.com/specify/specify7/blob/75d0412f92feaa9b9e92bc1c108d0e6db713f5e0/specifyweb/frontend/js_src/lib/components/DataModel/types.ts#L1-L22

In case the frontend types were added manually, can you autogenerate them with these steps?